packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.tsx

```diff
@@ -159,6 +160,7 @@ export class DetailsHeader extends BaseComponent<IDetailsHeaderProps, IDetailsHe
                 styles.cell,
                 styles.cellIsCheck,

```

Comment:
why wouldn't this style be defined in `styles.cellIsCheck` ? Why define a header specific rule in check itself?

Reply:
If it was part of `DetailsRowCheck`, it would need to be in `props`, so I tried to avoid it. Went ahead and added it anyway.